### DISCUSSION TITLE
esp to esp a2dp connect to mac address

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#928] [Q&A] [answered]
I want selected ESP only to connect  a2dp-SINK so do we have option to replace this line  [  a2dp_sink.start("AudioKit"); ] with mac address so that no other device mobile OR any other can connect to it  .

**Comment by pschatzmann** [comment, marked as answer]
Why don't you look at the API documentation ?
There is [even an example](https://github.com/pschatzmann/ESP32-A2DP/blob/main/examples/bt_music_sender_ssid_callback/bt_music_sender_ssid_callback.ino) for this!
